content/zh-cn/docs/reference/glossary/endpoint.md

```diff
@@ -2,49 +2,43 @@
 title: 端点（Endpoints）
 id: endpoints
 date: 2020-04-23
-full_link: 
+full_link: /zh-cn/docs/concepts/services-networking/service/#endpoints
 short_description: >
-  Service 的一个端点是实现该服务的 Pod 之一（或外部服务器）。
-
-aka:
+  表示 Service 端点的 API（已弃用）
 tags:
 - networking
 ---
 <!--
 title: Endpoints
 id: endpoints
 date: 2020-04-23
-full_link: 
+full_link: /docs/concepts/services-networking/service/#endpoints
```

Comment:
???
```suggestion
full_link: /zh-cn/docs/concepts/services-networking/service/#endpoints
```

Reply:
Oh, this is the commented original english text. `/zh-cn` has beed added on line 5.